packages/core-utils/src/types.js

```diff
@@ -243,6 +306,25 @@ export const transitIndexStopWithRoutes = PropTypes.shape({
   )
 });
 
+/**
+ * This models data about vehicle rental stations as obtained from various
+ * vehicle rental API endpoints from OTP.
+ */
+export const stationsType = PropTypes.arrayOf(
```

Comment:
Did you mean `stationType`? (It seems to be about one station as opposed to multiple.)

Reply:
No, it's specifically for multiple as indicated by the arrayOf propType.

Reply:
Oh I missed the inner array. Then do you want to do a type for one station, and have the stations props to be PropTypes.arrayOf(stationType)?